seedAlsoViewed.js

```diff
@@ -90,13 +93,14 @@ for (var i = 1; i < 101; i++) {
   var currentPrice = getCurrentPrice(oldPrice);
   var freeSheeping = getFreeShipping();
   var shippingCost = getShippingCost(freeSheeping);
+  var categoryId = getCategoryId();
 
-  seedAlsoViewedItems.push([id, image, title, oldPrice, currentPrice, freeSheeping, shippingCost]);
+  seedAlsoViewedItems.push([id, image, title, oldPrice, currentPrice, freeSheeping, shippingCost, categoryId]);
 }
 
 
 /// insert seed data
-var queryString = 'insert into alsovieweditems (id, image, title, oldprice, currentprice, freeshipping, shippingcost) values ?';
+var queryString = 'insert into alsovieweditems (id, image, title, oldprice, currentprice, freeshipping, shippingcost, categoryid) values ?';
```

Comment:
Does this single question mark denote shorthand for a single insertion for each of your columns? I've only seen this written so far to appear like: 

`'insert into alsovieweditems (id, image, title, oldprice, currentprice, freeshipping, shippingcost, categoryid) values (?, ?, ?, ?, ?, ?, ?, ?)';`

Reply:
You can see in my queryArgs I am sending an array of an array. I am doing multiple insertion query and my each element of queryArgs is an array with specified column values. Hence using single question mark to represent the same.